### DISCUSSION TITLE
"Moments: Error 1364: Field 'title' doesn't have a default value (create folder)"

### DISCUSSION THREAD

**Discussion by schellbell** [photoprism/photoprism#3384] [Q&A]
I have noticed that recently, PhotoPrism is doing something in the background (presumably relating to moments) that it is struggling with. While PhotoPrism appears to be working normally, a look at the logs shows the same issue being repeated every 20-30 minutes 
<img width="806" alt="image" src="https://user-images.githubusercontent.com/26511627/236688291-c32e135d-ffa0-4af9-b11c-236cbf420e14.png">
This is happening constantly throughout the day/night when I am not actively accessing or updating PhotoPrism. Does anyone know a) what PhotoPrism is trying to accomplish, and b) what I can do to resolve the error?

**Comment by graciousgrey** [comment]
There is a background job that  updates moments. We did not see those errors before.
What PhotoPrism version and database version are you using? Have you made manual changes to the database in the past or migrated from Sqlite?

**Comment by schellbell** [comment]
Hello! I'm currently using Build 230506-9de9a3540 and MariaDB. I had been having a different error previously (Moments and Calendar weren't populating at all) so last week I completely removed both PhotoPrism and its database in MariaDB and started from scratch. That resolved the original issue of the Calendar not populating, but it seems I have now run into this issue

**Reply by lastzero** [reply]
Which MariaDB version, using the default settings or with a custom config? Note that others don't see such errors, so the only thing I can possibly think of are uncommon folder names that leave the title field for folders in an empty state... it's really a lot of guesswork without knowing more.

**Reply by schellbell** [reply]
Apologies, I'm using MariaDB 10.6.12 with the default settings. 

As for the uncommon folder names, the whole library is sorted into Year > Month > Day folders (ie entirely numeric) 
![image](https://github.com/photoprism/photoprism/assets/26511627/4d138160-1fd8-45f4-944b-2e60e28d3dd5)

**Reply by lastzero** [reply]
I don't see how that would be a problem. Could you try if using MariaDB 10.11 makes a difference?

**Reply by schellbell** [reply]
Ok will do. I'm also assuming you mean 10.6.11?

**Reply by lastzero** [reply]
No, I mean the `mariadb:10.11` image as used in our config example:
https://dl.photoprism.app/docker/docker-compose.yml

**Reply by schellbell** [reply]
Alright! I managed to update to 10.11. It broke my database in the process for whatever reason, so I ended up doing a clean setup of both PhotoPrism and MariaDB. That being said, it has been running post-index for about 12 hours now and I haven't had those weird errors show up (yet)!
